### PR TITLE
Copy over Openshift Operations worker

### PR DESCRIPTION
**Pull request by agrare**
This moves the openshift operations worker from https://github.com/ManageIQ/topological_inventory-operations-openshift into this repo which will eventually become `topological_inventory-openshift`

### CODE REVIEW

**Comment by agrare** [comment]
@eclarizio can you take a look?  I had to disable the authentication retriever because there is no DB access in this repo

**Comment by eclarizio** [comment]
@agrare Everything seems ok to me, but I'm confused why this is being moved? Are we consolidating all of the "worker" type functionality into this repository? I figured it would be the other way around, moving the collection stuff into the "operations" repo since collecting seems like an operation to me.

**Comment by agrare** [comment]
@eclarizio this is eventually going to be just `topological_inventory-openshift` repo with all of the openshift workers here.

**Comment by eclarizio** [comment]
@agrare Sounds fine to me, then. 👍 I'll just wait until this is merged to apply the authentication retriever changes that use the internal API onto this repo.

File: lib/topological_inventory/openshift/logging.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module TopologicalInventory
+  module Openshift
+    class << self
+      attr_writer :logger
+    end
+
+    def self.logger
+      @logger ||= Logger.new(STDOUT, :level => Logger::INFO)
```
Comment (bdunne) [inline lib/topological_inventory/openshift/logging.rb]: I think this should use `ManageIQ::Loggers::Container` if possible.  https://github.com/ManageIQ/manageiq-loggers/blob/master/lib/manageiq/loggers/container.rb

**Comment by Fryguy** [comment]
@eclarizio Yes, we are going to rename this repo to `topological_inventory-openshift`, and it will house multiple things.  I did the renamespacing work in #26 , and now we just have to work out the deployment issues before we do the actual rename.

File: lib/topological_inventory/openshift/operations/core/retriever.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+module TopologicalInventory
+  module Openshift
+    module Operations
+      module Core
+        class Retriever
+          def initialize(id)
+            @id = id
+            uri = URI.parse(ENV["TOPOLOGICAL_INVENTORY_URL"])
+            TopologicalInventoryApiClient.configure do |config|
+              config.base_path = "#{ENV["PATH_PREFIX"]}/topological-inventory/v0.0/"
```
Comment (bdunne) [inline lib/topological_inventory/openshift/operations/core/retriever.rb]: Prefer v0.1, I see v0.0 as deprecated.  We won't be adding anything new to it.

**Comment by agrare** [comment]
@bdunne yeah there's still a lot to clean up, this is just to move it into the main repo and get the specs passing.

File: lib/topological_inventory/openshift/logging.rb (hunk 1 above)
Comment (agrare) [reply lib/topological_inventory/openshift/logging.rb]: :+1: will do in a follow-up

File: lib/topological_inventory/openshift/operations/core/retriever.rb (hunk 2 above)
Comment (agrare) [reply lib/topological_inventory/openshift/operations/core/retriever.rb]: :+1:

**Comment by agrare** [comment]
@Fryguy can you take a look?

**Pull request merged**